app/scripts/bump_version.groovy

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env groovy
+
+def int[] findVersions() {
+    int[] versions = [0, 0, 0]
+
+    File file = new File("versions.gradle")
+
+    def majorRE = file.text =~ /appVersionMajor = (\d+)/
+    if (majorRE.find()) {
+        versions[0] = Integer.valueOf(majorRE.group(1))
+    }
+
+    def minorRE = file.text =~ /appVersionMinor = (\d+)/
+    if (minorRE.find()) {
+        versions[1] = Integer.valueOf(minorRE.group(1))
+    }
+
+    def patchRE = file.text =~ /appVersionPatch = (\d+)/
+    if (patchRE.find()) {
+        versions[2] = Integer.valueOf(patchRE.group(1))
+    }
+
+    return versions
+}
+
+
+def String generateVersion(int[] versions) {
+
+    def content = """ext {
+	appVersionMajor = ${versions[0]}
+	appVersionMinor = ${versions[1]}
+	appVersionPatch = ${versions[2]}
+}
+"""
+
+    File file = new File("versions.gradle")
+    file.text = content
+
+    return content
+}
+
+
+def void commit(int[] versions) {
+
+    def version = "${versions[0]}.${versions[1]}.${versions[2]}"
+
+    def git_add = "git add versions.gradle"
+    def git_commit = "git commit -m 'Increase version'"
```

Comment:
The pipeline is configured to clean the workspace when started, so it is presumed to be update